lib/screens/routing_screen.dart

```diff
@@ -23,15 +23,20 @@ class _RoutingScreenState extends State<RoutingScreen> {
 
   @override
   void initState() {
-    super.initState();
+    super.initState(); 
+    Future.delayed(const Duration(seconds: 1), () {
+      PylonsWallet.instance.exists().then((walletExists) async {
```

Comment:
this is a repetition of the code line 34-36, is there a reason why we are checking if the wallet app exists twice?